src/darwin/Framework/CHIP/MTRCluster.h

```diff
@@ -26,7 +28,7 @@ MTR_DEPRECATED(
 typedef void (^SubscriptionEstablishedHandler)(void);
 
 typedef void (^MTRStatusCompletion)(NSError * _Nullable error);
-typedef void (^MTRSubscriptionEstablishedHandler)(void);
+typedef void (^MTRSubscriptionEstablishedHandler)(chip::SubscriptionId aSubscriptionId);
```

Comment:
This change is not ok at all.  This is changing public shipped API.  You can't do that on Darwin.

Reply:
removed darwin change, thanks